srcs/helpers/ft_free_split.c

```diff
@@ -23,4 +23,5 @@ void	ft_free_split(char **split)
 		i++;
 	}
 	free(split);
+	split = NULL;
```

Comment:
Is a common practice to assign a pointer to NULL after freeing the memory it points to, i.e., after calling free();. This is done to avoid dangling pointers, which are pointers that still hold the address of a memory location that has been freed.